extras/controllerClient/examples/example_rust/src/lib.rs

```diff
@@ -0,0 +1,108 @@
+// Copyright(C) 2023 NV Access Limited, Leonard de Ruijter
+// This file is covered by the GNU Lesser General Public License, version 2.1.
+// See the file license.txt for more details.
+
+mod bindgen;
+
+pub use bindgen::{error_status_t, wchar_t};
+use bindgen::{
+    nvdaController_brailleMessage, nvdaController_cancelSpeech, nvdaController_getProcessId,
+    nvdaController_setOnSsmlMarkReachedCallback, nvdaController_speakSsml,
+    nvdaController_speakText, nvdaController_testIfRunning, onSsmlMarkReachedFuncType,
+    SPEECH_PRIORITY, SYMBOL_LEVEL,
+};
+use windows::{
+    core::{Result, HSTRING},
+    Win32::Foundation::{ERROR_SUCCESS, WIN32_ERROR},
+};
+
+#[repr(u32)]
+#[derive(Debug, Copy, Clone, Hash, PartialEq, Eq)]
+pub enum SpeechPriority {
+    Normal = 0,
+    Next = 1,
+    Now = 2,
+}
+
+#[repr(i32)]
+#[derive(Debug, Copy, Clone, Hash, PartialEq, Eq)]
+pub enum SymbolLevel {
+    None = 0,
+    Some = 100,
+    Most = 200,
+    All = 300,
+    Char = 1000,
+    Unchanged = -1,
+}
+
+pub type OnSsmlMarkReached = onSsmlMarkReachedFuncType;
+
+fn is_succes(error: u32) -> Result<()> {
```

Comment:
Sadly you have a typo here. Maybe `to_result(status: u32)` would be more appropriate?

Reply:
Thanks, I'll rename accordingly.